drivers/ethernet/Kconfig.dsa

```diff
@@ -0,0 +1,36 @@
+# Distributed Switch Architecture [DSA] configuration options
+
+# Copyright (c) 2020 DENX Software Engineering GmbH
+#               Lukasz Majewski <lukma@denx.de>
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig NET_DSA
+	bool "Distributed Switch Architecture support"
+	depends on NETWORKING
+	help
+	  Enable Distributed Switch Architecture support. For now it only supports
+	  Kinetics ENET driver.
+
+if NET_DSA
+
+config DSA_KSZ8794
```

Comment:
add this config option in the commit that adds this driver

Reply:
OK.